### PR TITLE
[#61] Update to molecule 3.x

### PR DESCRIPTION
**Pull request by sorobon**
Update to molecule 3.x

### CODE REVIEW

**Comment by dortegau** [comment]
It would be nice to rename this role to `tomcat_role`:

https://galaxy.ansible.com/docs/contributing/creating_role.html#role-names

File: README.md [hunk 1]
```diff
@@ -26,6 +28,7 @@ For compatible Ansible versions check [.travis.yml](.travis.yml).
 Inventory destination should be a Debian environment.
 
 For testing purposes, [Molecule](https://molecule.readthedocs.io/) with Docker as driver and [Goss](http://goss.rocks) as verifier
+We need a system with java previously installed, we recommend the use of our role for that purpose: [idealista.java-role](https://github.com/idealista/java_role)
```
Comment (dortegau) [inline README.md:31]: It's better to use an impersonal language :)

File: molecule/default/tests/test_tomcat.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      - tomcat
+
+group:
+  tomcat:
+    exists: true
+
+port:
+  tcp:8080:
+    listening: true
+  tcp:8005:
```
Comment (dortegau) [inline molecule/default/tests/test_tomcat.yml:21]: Maybe the AJP port should be tested too :)

Comment (sorobon) [reply molecule/default/tests/test_tomcat.yml:21]: done

**Pull request merged**